README.md

```diff
@@ -0,0 +1,32 @@
+# ERC2048
+
+Inspired by [ERC404](https://github.com/Pandora-Labs-Org/erc404), ERC2048 is an experimental, mixed ERC20 / ERC721 implementation with native liquidity and fractionalization.
+
+In ERC404, there is a 1:1 mapping between fungible tokens (FT) and non-fungible tokens (NFT). This caused two challenging issues when using the ERC404 protocol:
+
+1. When transferring a large amount of tokens, the times of minting and burning NFTs equal to the token amount, it'll cost too much Gas. In the initial version of ERC404 code, it try to use whitelist to make it possible to transfer large amount, but it proved ineffective and caused a bug.
+2. With the 1:1 mapping between NFTs and tokens, different rarities of NFTs lose their value. Even if a green Pandora box has a lower probability of generation compared to a red Pandora box, they are both equal to 1 token, right?
```

Comment:
2. With the 1:1 mapping between NFTs and FTs, different rarities of NFTs lose their value. Even if a green Pandora box has a lower rarity compared to a red Pandora box, they are both equal to 1 Pandora.